python/compress-image/.gitignore

```diff
@@ -0,0 +1,4 @@
+code.tar.gz
+image_test/
+secret.py
```

Comment:
Consider including a `secret.py` file which shows what secrets can be set, but just set them to `""` or `None`